index.js

```diff
@@ -24,15 +24,28 @@ function browserSyncPlugin(options){
             }
         });
 
-        //Require options that are non-negotiable
-        bsOptions.middleware = function (req, res, next) {
-            metalsmith.build(function (err) {
+        var rebuild = function() {
```

Comment:
Thanks for taking the time to redo this using the new API! Could you please change this to be a function declaration instead of a variable?